os/tools/mkbootparam.py

```diff
@@ -103,12 +109,22 @@ def make_bootparam():
 
     with open(bootparam_file_path, 'wb') as fp:
         # Write Data
+        bootparam_version = 0
+        bootparam_format_version = 1.0
```

Comment:
And is there any special reason that `bootparam_version` is int type, and `bootparam_format_version` is float type?

Reply:
I changed a type of `bootparam_format_version` to int type.